src/govuk/components/skip-link/skip-link.js

```diff
@@ -0,0 +1,81 @@
+function SkipLink ($module) {
```

Comment:
I think we're missing some polyfill imports in this file… at least `Element.classList`, but possibly others?

(It'll still work in older browsers because of #674, unfortunately… not sure what the best way is to test this)

Reply:
Good catch, this is a bit of a blind spot 🤕  I've added the imports and tested this works by removing all the other components from `all.js` in checking that the skip link works in older IE and there are no errors in the console.